lib/index.js

```diff
@@ -162,3 +191,15 @@ Ramen.prototype.group = function(group) {
 
   window.Ramen.go();
 };
+
+Ramen.prototype.page = function() {
+  window.Ramen.go();
+};
+
+Ramen.prototype.track = function(track) {
+  window.Ramen.Api.track_named(track.event());
+};
+
+Ramen.version = function() {
```

Comment:
I'm reluctant to add a nonstandard method that we'll have to meticulously maintain or risk breaking the client script. Instead, can we disable the interval checking from within the `#initialize` method? Is it possible to expose that as a public API in the Ramen script?